src/components/ContactsList/ContactHeaderRow.jsx

```diff
@@ -1,9 +1,45 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { withStyles } from '@material-ui/core/styles'
 
-const ContactHeaderRow = props => <div className="divider">{props.header}</div>
+import Paper from 'cozy-ui/transpiled/react/Paper'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+const customStyles = theme => ({
+  paper: {
+    position: 'sticky',
+    zIndex: 1,
+    top: 0,
+    backgroundColor: 'var(--defaultBackgroundColor)',
+    height: '2rem',
+    display: 'flex',
+    alignItems: 'center',
+    textIndent: '2rem',
+    textTransform: 'uppercase',
+    [theme.breakpoints.down('sm')]: {
+      textIndent: '1rem'
+    }
+  },
+  typography: {
+    fontSize: '0.75rem'
```

Comment:
Why do we need a specific font-size here ? Are we missing a Typography variant in cozy-ui ?

Reply:
C'est un cas spécifique je pense, besoin d'un h6 en 12px (comme caption) ou d'un caption en bold (comme h6). J'ai changé pour un caption en bold, c'est interroge probablement moins.